### DISCUSSION TITLE
Install will not finish for securityonion 2.3.30

### DISCUSSION THREAD

**Discussion by byronsims** [Security-Onion-Solutions/securityonion#3299] [Unsupported Versions] [answered]
I am trying to setup a new distributed environment.  I'm creating the master server and the install will not finish.  I have done the steps found: https://github.com/Security-Onion-Solutions/securityonion/discussions/2285#discussioncomment-198643, but it has not helped.
[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/6086790/sosetup.log)

**Comment by TOoSmOotH** [comment, marked as answer]
Looking at your log its not able to reach the internet to download the containers. Something looks to be blocking it.

**Reply by byronsims** [reply]
Is it using a port beside 80 or 443?

**Reply by byronsims** [reply]
It seems like an issue with docker starting.  It was able to get three files in /root/sosigs.

**Reply by byronsims** [reply]
I figured out this initial issue...  Apparently I was running out of space.  I used an image provided by my system administrators and did not realize they had broken out /var into its own logical volume.

Thanks,

Byron
